### ISSUE TITLE
Допуск к защите в феврале

### ISSUE THREAD

**Issue by aakuptsov** [ReznikovRoman/Project#8]
Ничего не прислано. Нет допуска к защите в феврале.

**Comment by ReznikovRoman** [comment]
Александр Александрович, добрый вечер! Я прислал проект, классовые диаграммы и другие документы 29 января. Видео не удалось сделать по техническим причинам, сегодня я уже доделываю видео. 
Почему написано, что ничего не прислано?

Sent from Mail for Windows 10

From: Alexander Kuptsov
Sent: 30 января 2020 г. 19:47
To: ReznikovRoman/Project
Cc: Subscribed
Subject: [ReznikovRoman/Project] Допуск к защите в феврале (#8)

Ничего не прислано. Нет допуска к защите в феврале.

**Comment by aakuptsov** [comment]
Привет, Роман, 

самый последний коммит, который я вижу, датирован "4 months ago". Где "лежит" всё, что ты прислал?

**Comment by ReznikovRoman** [comment]
https://github.com/ReznikovRoman/Project_2nd_Wave
Я же подходил к вам лично с этим вопросом

Sent from Mail for Windows 10

**Comment by aakuptsov** [comment]
А, точно, прости. Я не исправил в своей сводной таблице. Всё ок. Но видео доделай, пожалуйста.

**Comment by ReznikovRoman** [comment]
Да, конечно. Презентацию я же могу доделать к самой защите? Мне просто на этих выходных некоторые администраторы напишут свои отзывы по поводу использования приложения

Sent from Mail for Windows 10

**Comment by aakuptsov** [comment]
Да, конечно, презентацию мы заранее не смотрим, поэтому делай спокойно к защите.

**Comment by ReznikovRoman** [comment]
Ясно, спасибо, буду делать.
Видео прикреплю сегодня вечером

Sent from Mail for Windows 10
